src/field/phone-number/phone_number_pane.jsx

```diff
@@ -29,13 +28,13 @@ export default class PhoneNumberPane extends React.Component {
         {header}
         <SelectInput
           icon={icon}
-          isValid={!isFieldVisiblyInvalid(lock, "location")}
+          isValid={!c.isFieldVisiblyInvalid(lock, "location")}
           name="location"
           placeholder=""
           label={humanLocation(lock)}
           onClick={() => startOptionSelection(l.id(lock), "location", "", icon)}
         />
-        <PhoneNumberInput ref="phoneNumberInput"
```

Comment:
Thanks @glena. @hzalaz said the exact same thing. It's from passwordless and we shouldn't care about it 😁